staging/src/k8s.io/apiserver/pkg/storage/etcd3/store_test.go

```diff
@@ -1182,6 +1182,7 @@ func TestListContinuation(t *testing.T) {
 }
 
 func testSetup(t *testing.T) (context.Context, *store, *integration.ClusterV3) {
+	LeaseReuseLength = 10
```

Comment:
comment on why we set this to 10.

Reply:
Anything smaller than 30 (minus a very small number) would work. I don't think it matters. Any suggestions?

Reply:
we shall comment on why it is 10 not 1000 for example. saying 30 is the default timeout for testing, so we cannot wait longer than that in a single time ... should be good enough.

Reply:
done